lib/Features/Common/Home/View/home_page.dart

```diff
@@ -63,84 +64,91 @@ class _HomePageState extends State<HomePage> {
     ));
   }
 
-  Padding _buildNavbar() {
-    return Padding(
-      padding: const EdgeInsets.symmetric(horizontal: 24),
-      child: Align(
-        alignment: Alignment.topCenter,
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: [
-            SettingButton(
-              onPressed: () {},
-              iconColor: _selectedSegment == 0 ? Colors.white : Colors.black,
-            ),
-            Container(
-              height: 40,
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: [
-                  GestureDetector(
-                    onTap: () {
-                      setState(() {
-                        _selectedSegment = 0;
-                      });
-                      Provider.of<CaptureProvider>(context, listen: false)
-                          .init();
-                    },
-                    child: Container(
-                      width: 57,
-                      height: 32,
-                      decoration: BoxDecoration(
-                        color: _selectedSegment == 0
-                            ? Colors.blue
-                            : Colors.grey[300],
-                        borderRadius: const BorderRadius.only(
-                          topLeft: Radius.circular(9),
-                          bottomLeft: Radius.circular(9),
+  Widget _buildNavbar() {
+    return Container(
+      color: ABColors.deepSea,
+      child: IntrinsicHeight(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 24),
+          child: Align(
+            alignment: Alignment.topCenter,
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                SettingButton(
+                  onPressed: () {},
+                  iconColor: _selectedSegment == 0 ? Colors.white : Colors.white,
```

Comment:
langsung aja gausah pake ternary
```suggestion
                  iconColor: Colors.white,
```